benchmark/compare.js

```diff
@@ -76,11 +77,9 @@ if (showProgress) {
       // Construct configuration string, " A=a, B=b, ..."
       let conf = '';
       for (const key of Object.keys(data.conf)) {
-        conf += ` ${key}=${JSON.stringify(data.conf[key])}`;
+        conf += ` ${key}=${inspect(data.conf[key])}`;
       }
       conf = conf.slice(1);
-      // Escape quotes (") for correct csv formatting
-      conf = conf.replace(/"/g, '""');
```

Comment:
How does using `inspect` prevent quotes completely from appearing? Could `data.conf[key]` not contain a quite itself?

Reply:
Seconding this question.

Reply:
Uh, you are right. I was a bit hasty here. I revert that change.